src/adapter/src/catalog/state.rs

```diff
@@ -946,7 +947,31 @@ impl CatalogState {
                     initial_as_of,
                 })
             }
-            Plan::CreateContinualTask(CreateContinualTaskPlan {}) => todo!("WIP"),
+            Plan::CreateContinualTask(CreateContinualTaskPlan {
+                desc,
+                continual_task,
+                ..
+            }) => {
+                // TODO(ct): Figure out how to make this survive restarts. The
+                // expr we saved still had the LocalId placeholders for the
+                // output, but we don't have access to the real Id here.
+                let optimized_expr = OptimizedMirRelationExpr::declare_optimized(
+                    mz_expr::MirRelationExpr::constant(Vec::new(), desc.typ().clone()),
+                );
```

Comment:
I'm probably missing a bit here, but why can't we re-optimize the raw SQL? Is it because they're self-referential? This is how we persist information about all other catalog items.